architecture/topics/aws_getting_started_development.adoc

```diff
@@ -0,0 +1,249 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/aws-getting-started.adoc
+
+[id='aws-getting-started-development-{context}']
+= Using the AWS Broker in a development deployment
+
+When you use the AWS Broker in development environment a you deploy the broker into an new {product-title} cluster using [CatASB](https://github.com/fusor/catasb) to deploy OpenShift, Service Catalog, and AWS Broker.
+
+link:https://github.com/fusor/catasb[CatASB], or CATalogASB, is a collection of Ansible playbooks that automate the creation of an {product-title} environment containing the Service Catalog and the AWS Broker.
```

Comment:
Why would we promote this over doing a typical OCP install? Why don't these scripts / tools work with our ansible playbooks?

Reply:
@sferich888 I am not sure that we are still supporting this method. I spoke with Jay McConnell, the SME in Amazon, they changed their process to on;y deploy the broker into an existing OpenShift cluster. I have an email out to @spahl but have not heard back.  
I probably should have removed these topics from the PR in the meantime.

Reply:
All catasb references should be removed.

Reply:
@jwmatthews I removed the files and references to catASB. Thank you for the clarification.